pkg/kubectl/cmd/util/shortcut_restmapper.go

```diff
@@ -98,9 +98,9 @@ func (e ShortcutExpander) RESTMappings(gk schema.GroupKind, versions ...string)
 	return e.RESTMapper.RESTMappings(gk, versions...)
 }
 
-// userResources are the resource names that apply to the primary, user facing resources used by
+// UserResources are the resource names that apply to the primary, user facing resources used by
```

Comment:
Explain why this is exposed.  Something like "Downstream composition that wants to build on the concept of 'all' for their CLIs need access to a current list of resources".  Otherwise it will be privatized again.